src/screens/ProfileScreen/elements/BottomSheetBio.js

```diff
@@ -25,12 +26,16 @@ const BottomSheetBio = React.forwardRef((props, ref) => {
             placeholder="Message prompt"
             keyboardAppearDelay={500}
             editable={!props.isOtherProfile}
+            maxLength={350}
           />
           <Text style={styles.description}>{props.value ? props.value.length : 0}/350</Text>
           {props.error ? <Text style={styles.errorText}>{props.error}</Text> : null}
         </View>
         {!props.isOtherProfile && (
-          <Button style={styles.button} textStyling={styles.textStyling} onPress={props.handleSave}>
+          <Button
+            btnStyle={styles.button}
+            textStyling={styles.textStyling}
+            onPress={props.handleSave}>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change from using a `style` prop to a `btnStyle` prop on the `Button` component seems fine as long as the `Button` component is updated accordingly to accept and use this new prop. Also, make sure all instances of the `Button` component throughout the codebase are updated to reflect this change.
```diff
- <Button style={styles.button} textStyling={styles.textStyling} onPress={props.handleSave}>
+ <Button btnStyle={styles.button} textStyling={styles.textStyling} onPress={props.handleSave}>
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->